src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        final int numberOfBalls = 3;
```

Comment:
did you read more information about constants here? -  https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
we should move it to the class area, and make it `static final`